### ISSUE TITLE
Set `admin_info` of `Translation` module

### ISSUE THREAD

**Issue by sveneberth** [viur-framework/viur-core#1114]
In the script and moduleconf module we can set the admin_info via config:
https://github.com/viur-framework/viur-core/blob/3183be76f2428141e934152319bda0abe3813c44/src/viur/core/config.py#L690-L700

But the translation module has already a default `admin_info`:
https://github.com/viur-framework/viur-core/blob/3183be76f2428141e934152319bda0abe3813c44/src/viur/core/modules/translation.py#L128-L143

How should we solve this? A modifier which is a method?

**Comment by phorward** [comment]
#1158 related to this. Is it still open?

**Comment by sveneberth** [comment]
> #1158 related to this. Is it still open?

When I wanted to adjust something in the admin_info of my project, I realised that I can do this for all modules via the config. But not for the translations. So I thought it would be a good possibility to extend this. But since the translations have a standard admin info, it's not that easy.
The only way would be to define a method here, which is then called by the admininfo and can make changes to the admininfo. but then, in principle, I can also assign the method directly to the module using monkey patching. So there is no real advantage to assigning it to the config instead of the module. Except that it is logically together.
long story short, I think we can just leave it as it was last implemented in #1158.

**Comment by phorward** [comment]
> So there is no real advantage to assigning it to the config instead of the module. Except that it is logically together. long story short, I think we can just leave it as it was last implemented in #1158.

I think, by directly assigning to `viur.core.modules.translation.Translation.adminInfo` or sub-classing it will be okay when another behavior is wanted. This should be done similar to `Script` and `ModuleConf`. We might remove the config for those and rely on the direct assignment or subclassing.
